frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -113,7 +113,7 @@ action.GitMachete.BaseResetToRemoteAction.info-dialog.title=Git Machete Reset In
 action.GitMachete.BaseResetToRemoteAction.info-dialog.message.HTML=\
   <html>\
     Git Machete Reset will perform a reset of local branch <b>{0}</b> \
-    so that it will point to the same commit as its remote branch <b>{1}</b><nobr/>.<br><br> \
```

Comment:
Huh maybe remove all my recently added `<nobr>` then while you're here? 😅 better to have some period or comma go to the next line, than have the line truncated like in your screen below 👇🏻

Reply:
_to ja cofam to co powiedziałem przed chwilą_